code/ship/ship.cpp

```diff
@@ -2681,6 +2684,9 @@ int parse_ship_values(ship_info* sip, const bool is_template, const bool first_t
 	// calculate the max speed from max_velocity
 	sip->max_speed = sip->max_vel.xyz.z;
 
+	if(optional_string("$Player Minimum Velocity:"))
+		stuff_vec3d(&sip->min_vel);
```

Comment:
The minimum velocity is specified for all axes but only the Z axis is actually enforced. It's fine to keep this for future expansion but the documentation should clearly state the currently only the Z axis value is actually used.